app/src/main/java/org/jellyfin/androidtv/util/apiclient/PlaybackHelper.java

```diff
@@ -58,27 +59,29 @@ public static void getItemsToPlay(final BaseItemDto mainItem, boolean allowIntro
                                 ItemFields.PrimaryImageAspectRatio,
                                 ItemFields.ChildCount
                         });
-                        query.setUserId(TvApp.getApplication().getCurrentUser().getId());
-                        get(ApiClient.class).GetItemsAsync(query, new Response<ItemsResult>() {
+                        get(ApiClient.class).GetEpisodesAsync(episodeQuery, new Response<ItemsResult>() {
                             @Override
                             public void onResponse(ItemsResult response) {
                                 if (response.getTotalRecordCount() > 0) {
+                                    boolean foundMainItem = false;
                                     for (BaseItemDto item : response.getItems()) {
-                                        if (item.getIndexNumber() > mainItem.getIndexNumber()) {
+                                        if (foundMainItem) {
                                             if (!LocationType.Virtual.equals(item.getLocationType())) {
                                                 items.add(item);
                                             } else {
                                                 //stop adding when we hit a missing one
                                                 break;
                                             }
+                                        } else if (item.getId() != null && item.getId().equals(mainItem.getId())) {
+                                            foundMainItem = true;
```

Comment:
This _should_ be doable with the api by setting the `StartItemId` but it seems that is not supported in the old apiclient. It maybe a good idea to add a TODO or FIXME comment here about it though.